logging/pulsar-beat-output/v0.1.0/pulsar-beat-output.md

```diff
@@ -0,0 +1,133 @@
+---
+description: The Pulsar Beat output plugin is a lightweight data shipper, written in Go, that you install on your servers to capture all sorts of operational data.
```

Comment:
Since `description` displays on the homepage as below
![image](https://user-images.githubusercontent.com/50226895/85945474-8f4ec000-b970-11ea-9dd7-b1c959d553b2.png)

Can we change `The Pulsar Beat output plugin is a lightweight data shipper, written in Go, that you install on your servers to capture all sorts of operational data.` to `Use Pulsar Beat output plugin to capture all sorts of operational data`?